test/specs/repo/special-property.ts

```diff
@@ -37,4 +37,25 @@ describe( 'Special:NewProperty', function () {
 
 		assert.strictEqual( numberOfPropertiesAfter, numberOfPropertiesBefore + 1 );
 	} );
+
+	it( 'Should be able to create a new property of datatype Item', async () => {
+		await SpecialNewProperty.open();
+
+		await SpecialNewProperty.labelInput.setValue( 'Cool Item label' );
+		await SpecialNewProperty.descriptionInput.setValue( 'Cool Item description' );
+		await SpecialNewProperty.aliasesInput.setValue(
+			'Great Item!|Greatest Item!'
+		);
+
+		await SpecialNewProperty.datatypeInput.click();
+		await browser.pause( 1 * 1000 );
+		await $( '.oo-ui-labelElement-label=Item' ).click();
```

Comment:
You can select the *Item* datatype with `wiki/Special:NewProperty?datatype=wikibase-item`.

I like the idea to actually use the UI to select the datatype, I do not like, that we add another wait for that. I'd suggest:
 - either just use the url again
 - or find a way to properly wait for the element you want to click on

Reply:
Yes, unless unless there is no other way, I think we shouldn't be using `browser.pause`. If we can do that through UI selection and await display of a UI element I also agree that would be best. 

In this case I would DRY this one up and use the existing "Should be able to add statement and reference to property" test iterate over `[ 'string', 'wikibase-item' ]`. Bonus: add another type or two to the array.

Reply:
You could use the url, yes, but I'm trying to match the test instructions, which do include clicking on the relevant type
I'll see about removing the pause

Reply:
Didn't update here, apparently, but I removed the pause